Added option to change word wrap width size.

marcovirtual opened the pull request:
`word_wrap` and `wrap_width` have been added to the `FontCycler.py` file.
`README.md` has also been updated to reflect the changes.

marcovirtual committed:
Added option to change word wrap width size.

word_wrap and wrap_width have been added to the FontCycler.py file.
README.md has also been updated to reflect the changes.

akshaykumar90 commented on README.md:
```diff
@@ -17,73 +17,75 @@ You can cycle through the fonts by pressing the `F10` key, and cycle backwards w
-      "Source Code Pro",
-      "Ubuntu Mono"
-    ]
+	"fonts_list":
+		[
+			"Andale Mono",
+			"Consolas",
+			"Courier",
+			"Courier New",
+			"Envy Code R",
+			"Inconsolata",
+			"Lucida Console",
+			"Menlo",
+			"Meslo LG S",
+			"Meslo LG M",
+			"Meslo LG L",
+			"Monaco",
+			"ProFontX",
+			"Source Code Pro",
+			"Ubuntu Mono"
+		]
```
Can you undo this whitespace change? Same with a couple more below.

marcovirtual replied:
Done! I fixed the whitespace change, added the updated PackageControl URL and also added some examples with proportional width fonts. Sorry if it's a little messy, I've never done a git push/pull before.

akshaykumar90 commented:
@marcovirtual The change looks good. Just a comment about whitespace.

marcovirtual committed:
Fix tab issues, add var width fonts, fix URL

Fixed whitespace tab issues, added some proportional width font examples, and updated PackageControl URL.

marcovirtual committed:
Fix tab issues, add var width fonts, fix URL

Fixed whitespace tab issues, added some proportional width font examples, and updated PackageControl URL.
